J/JSBSim/build_tarballs.jl

```diff
@@ -0,0 +1,58 @@
+using BinaryBuilder
+
+name = "JSBSim"
+version = v"1.1.6"
+julia_version = v"1.5.3"
+
+# Collection of sources required to build JSBSim
+sources = [
+    GitSource("https://github.com/bcoconni/jsbsim.git",
+              "d699c3bcd90bf9ce2a64c1fe3091fcf39714ccfd"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/jsbsim
+mkdir build && cd build
+FLAGS=()
+if [[ "${target}" == *-mingw* ]]; then
+    FLAGS+=(-DCMAKE_CXX_FLAGS_RELEASE="-D_POSIX_C_SOURCE")
+fi
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_FIND_ROOT_PATH=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DBUILD_DOCS=OFF \
+    -DBUILD_PYTHON_MODULE=OFF \
+    -DBUILD_JULIA_PACKAGE=ON \
+    -DJulia_PREFIX=${prefix} \
+    "${FLAGS[@]}" \
+    ..
+cmake --build . --target JSBSimJL -- -j${nproc}
+install_license $WORKSPACE/srcdir/jsbsim/COPYING
+cp julia/*JSBSimJL*.$dlext $libdir/.
+cp ../julia/JSBSim.jl $prefix/.
+"""
+
+ARGS
```

Comment:
What's this line about?

Reply:
I haven't spent much time investigating the issue, but without it...`ARGS` seems to disappear until the end of the file...  Maybe it's "consumed" by the `include`?

Reply:
Huh, OK, that's weird... I haven't noticed such issues with the other files using cxxwrap, but then again I am not 100% sure I would have noticed? Odd

Reply:
I was also puzzled because it works in polymake.  I didn't have the time to understand why `ARGS` would become undefined after the include and just dropped this workaround.  _...one day we'll figure out what's going on..._

Reply:
Fixed by #2593 
```suggestion
```